### DISCUSSION TITLE
Home Link gets a 404 error code

### DISCUSSION THREAD

**Discussion by talkativemom** [community/community#21638] [Other Feature Feedback, Questions, & Ideas]
<p>I have created my website and everything looks pretty good. Most things are working the way I want it too but then when i click on any of the other navigation links they work. But if I then click back on the home navigation link it gives me a 404 error code. I have tried for days through post on github and youtube to fix it but I am literally at a lost. Here is where my code is hosted: <a href="https://talkativemom.github.io/morganfamilywebsite/" class="inline-onebox" rel="noopener nofollow ugc">Morgan Family</a></p>
<p>GitHub - talkativemom/morganfamilywebsite</p>

**Comment by kingthorin** [comment, marked as answer]
<p>Looks like you have a typo in the home link on other pages:<br>
<code>&lt;li class="nav-item"&gt; &lt;a class="nav-link" href="index.com"&gt;Home&lt;/a&gt; &lt;/li&gt;</code></p>
<p>Should be <code>index.html</code> not <code>index.com</code></p>

**Comment by talkativemom** [comment]
<p>OMG Thank you so much. I have stared at this code for 2 weeks trying to figure out what was wrong with it.</p>
